tests/unit_tests/test_obstacle_map.py

```diff
@@ -17,13 +17,16 @@ def setUp(self):
 		self.obstacle_map3 = ObstacleMap(self.test_drone_point3,self.test_fly_zones2)
 
 	def test_add_obstacle(self):
-		"""	
+		"""
 		 test if the last obstacle in obstacle_map is the added object
 		"""
-		
+
 		self.test_obstacle = StationaryObstacle(numpy.array([1,1,1]),2,1)
 		self.obstacle_map1.add_obstacle(self.test_obstacle)
-		
+
+		#self.test_obstacle2 = StationaryObstacle(numpy.array([1,1,1]),4,1)
+		#self.test_obstacle3 = StationaryObstacle(numpy.array([1,1,1]),2,1)
+
 		self.assertEqual(self.obstacle_map1.get_obstacles(),self.test_obstacle)
```

Comment:
There should be a space after commas to keep the code readable.

Reply:
done